src/status_im/ui/components/copyable_text.cljs

```diff
@@ -117,8 +117,8 @@
            [copy-action-visual-cue anim-opacity anim-y width cue-atom]
            [react/touchable-highlight
             {:active-opacity (if @cue-atom 1 0.85)
-             :underlay-color colors/black
+             :underlay-color "rgba(255, 255, 255, 0.2)"
```

Comment:
all new colors should be in quo2 colors ns

Reply:
there are some colors already in this file that rely on quo/colors ( the old design system) I am working to move them to quo2 as well